Rlabkey/DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: Rlabkey
-Version: 2.8.3
-Date: 2022-02-24
+Version: 2.8.4
+Date: 2022-03-24
```

Comment:
the same date and version change should be made in the `Rlabkey-package.Rd` file at lines 21 and 22.